mxabierto/dgm
# Creates App of 'Atlas Turistico'

*** HOW TO TEST

1. Go to Apps Menu and check for this

![screen shot 2015-09-17 at 6 02 13 pm](https://cloud.githubusercontent.com/assets/2633527/9948651/97ebf212-5d6a-11e5-81aa-6880ab5951bf.png)

2. Click on "Atlas Turístico de México" link and check for this ( there should be 3 images on the carrousel )

![screen shot 2015-09-17 at 5 43 25 pm](https://cloud.githubusercontent.com/assets/2633527/9948660/b3a967a0-5d6a-11e5-814d-1f6d81a94c83.png)
